### ISSUE TITLE
Getting the Client IP Address Doc Update 

### ISSUE THREAD

**Issue by eabquina** [pantheon-systems/documentation#3690]
Re: [Getting the Client IP Address](https://pantheon.io/docs/client-ip/)

Priority (Low):

## Issue Description:
The customer claims that this method will not be compatible because of Varnish cache.  

```
1. Use the system environment variable $_SERVER["REMOTE_ADDR"]. One benefit is that on Pantheon this takes into account if the X-Forwarded-For header is sent in cases when a request is filtered by a proxy.
2. Use Drupal’s ip_address() function. The function leverages the $_SERVER["REMOTE_ADDR"] variable to get the client IP, as well as parse other meta information like the remote proxy IP (if available), trim the forwarded IPs if they contain any additional commas or spaces, and filter out any untrusted IPs.
```

## Suggested Resolution
Provide comment FAQ that this will be ok with caching layer

**Comment by alexfornuto** [comment]
@eabquina a few questions:

1)
> The customer claims that this method 

Which of the two methods listed is this referring to? Or is it both?

2) Have you tested this?

3) Are you saying that we need to make an addition to this doc saying that the doc does indeed work even if you (the reader) thinks it may not?

**Comment by alexfornuto** [comment]
Marking this as closed by #4134, please re-open with the requested additional details if this is not a full resolution.
